robottelo/performance/pulp.py

```diff
@@ -72,19 +74,19 @@ def get_enabled_repos(org_id):
         """
         LOGGER.info('Searching for enabled repositories by hammer CLI:')
 
-        result = Repository.list(
-            {'organization-id': org_id},
-            per_page=False
-        )
-
-        if result.return_code != 0:
+        try:
+            result = Repository.list(
+                {'organization-id': org_id},
+                per_page=False,
```

Comment:
We may improve this on the future, as is it no intuitive. Maybe a rename like auto_per_page=True and then you can make it False.